pkg/sloop/webserver/webfiles/resource.html

```diff
@@ -8,9 +8,9 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <link rel='shortcut icon' type='image/x-icon' href='/webfiles/favicon.ico'/>
-    <link rel="stylesheet" type="text/css" href="/webfiles/sloop.css">
-    <link rel="stylesheet" type="text/css" href="/webfiles/resource.css">
+    <link rel='shortcut icon' type='image/x-icon' href='webfiles/favicon.ico'/>
```

Comment:
it looks like all the other `.html` files are getting a script inserted - not this one?

Reply:
good observation :) yes resource.html is within main page. As the base href was set for the main page...its not needed by resource.html